docs/source/user_guide/micromamba.rst

```diff
@@ -117,7 +117,9 @@ Specification files
 
 The ``create`` syntax also allows you to use specification or environment files (also called *spec files*) to easily re-create environments.
 
-The three supported syntaxes are: yaml, txt and explicit spec files.
+The supported syntaxes are:
+
+.. contents:: :local:
```

Comment:
This generates a dynamic list of all the syntaxes in this doc. It's like a mini table-of-contents for the sub-headings of this heading.